search-service/src/test/kotlin/com/egm/stellio/search/authorization/SubjectReferentialServiceTests.kt

```diff
@@ -447,4 +448,55 @@ class SubjectReferentialServiceTests : WithTimescaleContainer {
                 fail("it should have returned a ResourceNotFoundException exception")
             })
     }
+
+    @Test
+    fun `it should get global roles for user with no role`() = runTest {
+        val userAccessRights = SubjectReferential(
```

Comment:
call this variable `subjectReferential`, not `userAccessRights` (and same in the two other tests)